tools/topology/topology2/include/pipelines/cavs/wov-detect.conf

```diff
@@ -57,6 +57,7 @@ Class.Pipeline."wov-detect" {
 		# to the host copier. The virtual widget does not exist in
 		# the firmware.
 		virtual."virtual.detect_sink" {
+			name "virtual.detect_sink"
```

Comment:
virtual. 0 { on line 69 please

Reply:
@libinyang can you respond to this change ? It looks different to the suggestion from @ranj063 ?

Reply:
1 works too @lgirdwood.